plates/dimensions.py

```diff
@@ -0,0 +1,23 @@
+available_dimensions = ((4, 6), (8, 12), (16, 24))
```

Comment:
Shouldn't be here, right? It makes this a global variable visible outside of `Dimensions`